tasks/run_cd4pe_job.rb

```diff
@@ -271,8 +271,8 @@ def send_job_output_to_cd4pe(output)
 
     begin
       response = client.make_request(:post, api_endpoint, payload.to_json)
-      if (response.code != 200)
-        @logger.log("Unable to send logs directly to CD4PE. Printing logs to std out.")
+      if (response.code != '200')
```

Comment:
We shouldn't be trying to do comparassions against strings or ints, and instead use ruby's methods for this, like we do on line 234.

```
if(!response.is_a? Net::HTTPSuccess)
```

See examples here:

https://ruby-doc.org/stdlib-2.7.1/libdoc/net/http/rdoc/Net/HTTP.html#class-Net::HTTP-label-Setting+Headers

and

https://ruby-doc.org/stdlib-2.7.1/libdoc/net/http/rdoc/Net/HTTP.html#class-Net::HTTP-label-Following+Redirection

Reply:
that is....an excellent point.